src/shared/components/deletion-confirmation/DeletionConfirmation.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+
+import { PrimaryButton } from 'src/shared/components/button/Button';
+
+import styles from './DeletionConfirmation.scss';
+
+export type DeletionConfirmationProps = {
+  warningText: string;
+  confirmText: string;
+  cancelText: string;
+  contentAlignRight?: boolean;
```

Comment:
`contentAlignRight?: boolean;` -> `alignContent: 'left' | 'right'`

...

```
const containerClass = classNames(
    styles.deleteConfirmationContainer,
   {[styles.contentAlignRight]: props.alignContent === 'right' },
    props.className
);
```

Reply:
so my thinking to this is by default the content will be aligned left and hence no need to specify this. And hence making the alignRight optional, its whether you want to align it right or not. Less props to pass through when you need a simple component.

Reply:
Yes, I know what you mean.

For my part, I though the name `contentAlignRight` slightly odd; and wondered what if we thought of this property in the same way we think about CSS properties, for which there is a default value (in this case, "left"); and other values (in this case, "right"). Passing the value "left" won't do anything; whereas passing the value "right" would change the arrangement of the element.

But I understand the reluctance to having a value that doesn't do anything. Let's see what the rest of the team thinks at scrum.

Reply:
I think `alignContent` would be a better option.

Reply:
Looking at the css [here](https://github.com/Ensembl/ensembl-client/pull/828/files#diff-7fce8969ad25904d78fb32760364f592a85b2b3328655b938dc54e989528e940R12), I wonder if this should be called `justifyContent` instead of `alignContent `?

Reply:
Again i am not sure to me `alignContent` is clearer than `justifyContent`

Reply:
Agreed to go with `alignContent` since it describes what we are trying to achieve. `justifyContent` is specific to css and also with display flex.